### PR TITLE
 feat: allow user to complete holds_up action type

### PR DESCRIPTION
**Pull request by juliano-quatrin-nunes**
Preview: 
image

### CODE REVIEW

File: apps/govquests-api/rails_app/app/services/action_tracking/op_token_balance.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          params: [
+            {
+              to: "0x4200000000000000000000000000000000000042",
+              data: "0x70a08231000000000000000000000000#{clean_address}"
+            },
+            "latest"
+          ],
+          id: 1
+        }.to_json,
+        headers: {"Content-Type" => "application/json"}
```
Comment (ribeirojose) [inline apps/govquests-api/rails_app/app/services/action_tracking/op_token_balance.rb:22]: 🪄

**Review by ribeirojose — APPROVED** [review]
only `#must` missing imo is this https://github.com/bleu/op-govquests/pull/78/files#r1850744054

File: apps/govquests-api/rails_app/app/graphql/mutations/start_action_execution.rb [hunk 2]
```diff
@@ -39,6 +39,10 @@ def resolve(quest_id:, action_id:, action_type:, send_email_verification_input:
         {
           address: current_user.address
         }
+      when "holds_op"
```
Comment (devjoaov) [inline apps/govquests-api/rails_app/app/graphql/mutations/start_action_execution.rb:42]:
```suggestion
      case input
when "ens", "wallet_verification", "verify_delegate_statement", "verify_delegate", "verify_first_vote", "holds_op"
  { address: current_user.address }
when "send_email"
  { email: send_email_verification_input&.email }
else
  {}
end
```

**Review by devjoaov — APPROVED** [review]

File: apps/govquests-api/govquests/action_tracking/lib/action_tracking/strategies/holds_op.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        start_data[:address].present?
+      end
+
+      def can_complete?
+        start_data_valid?
+      end
+
+      private
+
+      def has_op_tokens?
```
[the remark this answers is not in the document]
Comment (juliano-quatrin-nunes) [reply apps/govquests-api/govquests/action_tracking/lib/action_tracking/strategies/holds_op.rb:22]: Yep, I'll fix it 😬

**Pull request merged**